src/main/java/seedu/duke/commands/DeleteCommand.java

```diff
@@ -71,6 +71,9 @@ public CommandResult execute(ModuleList moduleList, Configuration configuration)
      */
     public void deleteModule(ModuleList moduleList) throws NoSuchModuleException {
         Module targetModule = moduleList.getModule(moduleCode);
+        if (Objects.isNull(targetModule)) {
```

Comment:
would throwing `NoSuchModuleException` in `getModule` instead of returning null be more concise since theres no need to check for null all the time?

Reply:
That is actually a good point, but there would need to be refactoring for every other Command that uses getModule as well. Might have to discuss with the others. I think the original idea was that the logic for validation should be done by Command instead of ModuleList, but that is not the case for TaskList. (Maybe I remembered it wrong)

Reply:
I'll do this in a different PR